PlaybackLab/ControlsLockScreenSampleApp/ControlsLockScreenSampleApp/Views/PlayerViewController.swift

```diff
@@ -0,0 +1,172 @@
+      // This is not how this should be implemented in production - In production, you should implement your own OOSecureURLGenerator
+      // which contacts a server of your own, which will help sign the url with the appropriate API Key and Secret
+      options.secureURLGenerator = OOEmbeddedSecureURLGenerator(apiKey: apiKey, secret: apiSecret)!
+      player = OOOoyalaPlayer(pcode: option.pcode, domain: option.domain, embedTokenGenerator: option.embedTokenGenerator!, options: options)
+    }
+    else {
+      player = OOOoyalaPlayer(pcode: option.pcode, domain: option.domain)
+    }
+    player.setEmbedCode(option.embedCode)
+    
+    addObservers()
+    setupCommandCenter()
+    setupPlayingInfoCenter()
+  }
+  
+  override func viewDidAppear(_ animated: Bool) {
+    super.viewDidAppear(animated)
+    player.play()
+  }
+  
+  func addObservers() {
+    NotificationCenter.default.addObserver(self, selector: #selector(applicationDidEnterBackground(_:)), name: NSNotification.Name.UIApplicationDidEnterBackground, object: nil)
+    NotificationCenter.default.addObserver(self, selector: #selector(playerStateChange(_:)), name: NSNotification.Name.OOOoyalaPlayerStateChanged, object: nil)
+  }
+  
+  func setupCommandCenter() {
+    remoteCommandCenter.playCommand.isEnabled = true
+    remoteCommandCenter.playCommand.addTarget(self, action: #selector(playPauseCommand))
+    
+    remoteCommandCenter.pauseCommand.isEnabled = true
+    remoteCommandCenter.pauseCommand.addTarget(self, action: #selector(playPauseCommand))
+    
+    remoteCommandCenter.skipBackwardCommand.isEnabled = true
+    remoteCommandCenter.skipBackwardCommand.preferredIntervals = [15]
+    remoteCommandCenter.skipBackwardCommand.addTarget(self, action: #selector(skipBackwardCommand(_:)))
+    
+    remoteCommandCenter.skipForwardCommand.isEnabled = true
+    remoteCommandCenter.skipForwardCommand.preferredIntervals = [15]
+    remoteCommandCenter.skipForwardCommand.addTarget(self, action: #selector(skipForwardCommand(_:)))
+    
+    remoteCommandCenter.changePlaybackPositionCommand.isEnabled = true
+    remoteCommandCenter.changePlaybackPositionCommand.addTarget(self, action: #selector(changePlaybackPositionCommand(_:)))
+  }
+  
+  func setupPlayingInfoCenter() {
+    // Set properties of the asset to be shown in the screen (no required).
+    var nowPlayingInfo: [String: Any] = [MPMediaItemPropertyTitle: option.title,
+                                         MPMediaItemPropertyArtist: "Ooyala",
+                                         MPMediaItemPropertyAlbumTitle: "Controls Lock Screen"
+    ]
+    
+    // Set albumArt to show in the screen if image is available
+    // The albumArt needs to be an UIImage, the image can be added in the assets of the project or retrieve the image using an URL
+    if let imageData: NSData = NSData(contentsOf: option.thumbnailURL), let image: UIImage = UIImage(data: imageData as Data) {
```

Comment:
no need to set types on the left side of the assignments.

Reply:
Done